src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -1456,14 +1455,10 @@ class CreateTableStatementSegment(ansi.CreateTableStatementSegment):
         Sequence(
             Bracketed(
                 Delimited(
-                    OneOf(
-                        Ref("TableConstraintSegment"),
-                        Ref("ColumnDefinitionSegment"),
-                    ),
+                    Ref("ColumnDefinitionSegment"),
                     allow_trailing=True,
                 )
             ),
-            Ref("CommentClauseSegment", optional=True),
```

Comment:
This hunk is a modification of CREATE TABLE.
According to https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#alter_table_add_column_statement, the column name and schema of ALTER TABLE ADD COLUMN  statement use the same syntax as CREATE TABLE, so removed non-existent syntax to ensure symmetry.

Reply:
Agree, coulnd't find any reference to them in the create table docs